File: rasterizer-dotnet-examples-csharp/PngImageExample.cs

using ceTe.DynamicPDF.Rasterizer;

namespace rasterizer_dotnet_csharp
{
    class PngImageExample
    {
        public static void Run()
        {
            DefaultName();
            ManualName();
        }

        public static void DefaultName()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"));
            rasterizer.Draw(Program.GetOutputDocPath("png-output-example.png"), ImageFormat.Png, ImageSize.Dpi72);
        }

        public static void ManualName()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"));

            for (int i = 0; i < rasterizer.Pages.Count; i++)
            {
                rasterizer.Pages[i].Draw(Program.GetOutputDocPath("manual-output" + i + ".png"), ImageFormat.Png, ImageSize.Dpi72);
            }
        }

    }
}

File: rasterizer-dotnet-vb/PngImageExample.vb
Imports ceTe.DynamicPDF.Rasterizer

Class PngImageExample

    Public Shared Sub Run()
        DefaultName()
        ManualName()
    End Sub

    Public Shared Sub DefaultName()
        Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"))
        rasterizer.Draw(Program.GetOutputDocPath("png-output-example.png"), ImageFormat.Png, ImageSize.Dpi72)
    End Sub

    Public Shared Sub ManualName()
        Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"))

        For i As Integer = 0 To rasterizer.Pages.Count - 1
            rasterizer.Pages(i).Draw(Program.GetOutputDocPath("manual-output" & i & ".png"), ImageFormat.Png, ImageSize.Dpi72)
        Next
    End Sub
End Class
